spec/services/encryption/encryptors/session_encryptor_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+
+describe Encryption::Encryptors::SessionEncryptor do
+  let(:plaintext) { '{ "foo": "bar" }' }
+
+  before do
+    described_class.instance_variable_set(:@user_access_key_scrypt_hash, nil)
+  end
+
+  describe '#encrypt' do
+    it 'returns encrypted text' do
+      ciphertext = subject.encrypt(plaintext)
+
+      expect(ciphertext).to_not eq(plaintext)
```

Comment:
Here and below, we test that the encrypted text is different than the plain text AND that `decrypt` does the opposite of `encrypt`. However, we're not actually testing that its actually *encrypting* or *decrypting* anything, only that its changing it. It looks like the actual encryption is tested in the `UserAccessKey` spec so I don't think we need to duplicate that, however, if might be more useful to verify that encryption is actually taking place, perhaps by verifying that the results are what we get from `UserAccessKeyEncryptor` and we may want to update the spec there to test that we're getting the result from `UserAccessKey`...